lib/github.js

```diff
@@ -63,9 +63,10 @@ Organization.prototype.getRepositories = function(callback) {
 
     request.get(options, function (error, response, body) {
 
-        if (error) {
-            cli.error(error);
-            callback.call(this, error);
+        if (error || response.statusCode != 200) {
+            var message = (error || "") +
```

Comment:
But before calling `cli.error` we have to check if there was error. I mean error could be false, but `response.statusCode != 200`.